spinalcordtoolbox/scripts/sct_process_segmentation.py

```diff
@@ -320,27 +369,60 @@ def main(argv=None):
                                          angle_correction=angle_correction,
                                          param_centerline=param_centerline,
                                          verbose=verbose)
+    if fname_pmj is not None:
+        im_ctl, mask, slices, centerline = get_slices_for_pmj_distance(fname_segmentation, fname_pmj,
+                                                                       distance_pmj, extent_mask,
+                                                                       param_centerline=param_centerline,
+                                                                       verbose=verbose)
```

Comment:
Maybe we could add a comment in the main help description to indicate what the correct usage is supposed to be:

```diff
            "Compute the following morphometric measures based on the spinal cord segmentation:\n"
            "  - area [mm^2]: Cross-sectional area, measured by counting pixels in each slice. Partial volume can be "
            "accounted for by inputing a mask comprising values within [0,1].\n"
            "  - angle_AP, angle_RL: Estimated angle between the cord centerline and the axial slice. This angle is "
            "used to correct for morphometric information.\n"
            "  - diameter_AP, diameter_RL: Finds the major and minor axes of the cord and measure their length.\n"
            "  - eccentricity: Eccentricity of the ellipse that has the same second-moments as the spinal cord. "
            "The eccentricity is the ratio of the focal distance (distance between focal points) over the major axis "
            "length. The value is in the interval [0, 1). When it is 0, the ellipse becomes a circle.\n"
            "  - orientation: angle (in degrees) between the AP axis of the spinal cord and the AP axis of the "
            "image\n"
            "  - solidity: CSA(spinal_cord) / CSA_convex(spinal_cord). If perfect ellipse, it should be one. This "
            "metric is interesting for detecting non-convex shape (e.g., in case of strong compression)\n"
            "  - length: Length of the segmentation, computed by summing the slice thickness (corrected for the "
            "centerline angle at each slice) across the specified superior-inferior region.\n"
+           "
+           "To select the region to compute metrics over, specify any of the following options:
+           "   - '-z': Select axial slices based on slice index.
+           "   - '-pmj' + '-distance' + '-extent': Select axial slices based on distance from pontomedullary junction.
+           "   - '-vert' + '-vertfile': Select a region based on vertebral levels, instead of individual slices.

```

Just so no one accidentally mixes and matches the mutually-exclusive arguments groups.

Reply:
yes good idea! There is also `-perlevel` that can be added with option 3, also I think `-perslice` can be mixed with `-z`, so many options 😄 we should add them too I think?

Reply:
I added the descriptions in https://github.com/spinalcordtoolbox/spinalcordtoolbox/pull/3478/commits/3dd2374d161ffad8b3a3aa1805baa2e6fd7524c1